app/serializers/encryptable/credentials_serializer.rb

```diff
@@ -1,7 +1,18 @@
 # frozen_string_literal: true
 
 class Encryptable::CredentialsSerializer < EncryptableSerializer
-  attributes :cleartext_username, :cleartext_password, :created_at, :updated_at
+  attributes :cleartext_username, :cleartext_password, :cleartext_token, :cleartext_pin,
+             :cleartext_email, :cleartext_custom_attr, :created_at, :updated_at
+
+  def cleartext_custom_attr
```

Comment:
can't we just have a separat attribute called `cleartext_custom_attr_label` for the label?
so we would not have to handle the cleartext value differently for the custom attr.
I don't like the idea of having a special structure here for the custom attr.